### DISCUSSION TITLE
[Help] Can zk resolve symbolic links?

### DISCUSSION THREAD

**Discussion by on3iro** [zk-org/zk#337] [Help] [answered]
### Verify

- [X] I searched the existing discussions for help

### How can we help you?

Quick question: is there a way to make zk resolve symlinks? I currently have a directory containing some notes as well as a symlinked subdirectory also containing notes. I want to combine both into a single notebook, but keep there "physical" location separated. Is there a way to do this?

Thanks in advance

**Comment by mickael-menu** [comment]
I think this should already be possible, have you tried?

**Reply by on3iro** [reply]
Yeah I did. I created a directory **A** which contains a regular directory **B** as well as a symbolic linked directory **C**. 
I initialized `zk` iniside **A**. When I now run `zk edit -i` I find all notes from **B** but none of **C**. (I am on OSX, btw.)

**Reply by mickael-menu** [reply, marked as answer]
Hmm yes, it looks like it works for symlinks of individual notes, but not for directories.

Maybe something to implement here, if you feel like it: https://github.com/mickael-menu/zk/blob/072fae2f6ce3f444bdb8cfc0be74b25740d1ed55/internal/util/paths/walk.go#L13

**Reply by on3iro** [reply]
Ah too bad, thanks for the answer. I'll have to checkout `go` then :D

Would you mind also giving me a quick answer on this one, please: https://github.com/mickael-menu/zk/discussions/336?

And on a side note: is there a way to quickly index existing tags from markdown files? :)

(Sorry for all these questions, but I am currently checking out zk and debating, if it could enhance/replace my current workflow)

**Reply by mickael-menu** [reply]
>And on a side note: is there a way to quickly index existing tags from markdown files?

This should be done automatically when running `zk list` the first time. You can try running `zk index --force` too.

**Reply by on3iro** [reply]
ah, yes that worked - awesome! :)
